kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
I'm obviously biased (I maintain kafka-python), but yes the kafka world has 2 functional python clients. The client apis are definitely different, the implementations are different, and the driver designs are different.

Even if you decide to ship w/ pykafka here, I still would like to use kafka-python w/ my teams that use celery at work. And I would prefer to not ship a second-class plugin even if it is not included in the official distribution.